rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -2111,7 +2116,7 @@ def run():
                                            'AW4_WGS_WORKFLOW',
                                            'AW5_ARRAY_MANIFEST',
                                            'AW5_WGS_MANIFEST',
-                                           'AW2F_MANIFEST'
+                                           'AW2F_MANIFEST',
```

Comment:
haha, haven't we fixed this?

Reply:
I keep fixing it locally and then not comitting it.